fvm/transactionInvoker.go

```diff
@@ -256,50 +256,45 @@ var setAccountFrozenFunctionType = &sema.FunctionType{
 }
 
 func valueDeclarations(env Environment) []runtime.ValueDeclaration {
-	var predeclaredValues []runtime.ValueDeclaration
-
-	if env.AccountFreezeEnabled() {
```

Comment:
The initial Idea was to have `setAccountFrozen` available as a function only if the authorizer is the service account, but I think this got lost along the way.

With the current implementation `setAccountFrozen` is available as a function to everyone, but will panic if called for everyone except the service account. 

@turbolent @ramtinms do you see any problems with this.

Reply:
my personal preference is to move this definition onto cadence side, and define it as part of the standard lib (and expose a SetAccountFrozen method to the environment interface).  I think it's fine to expose this function to everyone with the understanding that calling will fail if you're not the service account.

Reply:
This is very bad idea in my opinion, I commented something related to this today: https://github.com/onflow/cadence/issues/331#issuecomment-1208339185

I think moving environment functions to Cadence side is very bad idea. Cadence should be the language, fvm / or other environment hosting Cadence should provide the functions depending on the environment.